src/main/java/wooteco/subway/ui/LineController.java

```diff
@@ -0,0 +1,51 @@
+package wooteco.subway.ui;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.service.LineService;
+
+import java.net.URI;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+
+    private final LineService service;
+
+    public LineController(LineService service) {
+        this.service = service;
+    }
+
+    @PostMapping()
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse lineResponse = service.save(lineRequest);
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId())).body(lineResponse);
+    }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        return ResponseEntity.ok().body(service.findAll());
+    }
```

Comment:
```suggestion
    public List<LineResponse> showLines() {
        return service.findAll();
    }
```

상태 코드가 200 OK 라면 조금 더 간단하게 표현할 수 있습니다.

Reply:
와 이렇게 표현해도 되는군요! 몰랐습니다! 덕분에 코드가 더욱 간결해졌습니다. 감사합니다 🙇🏻‍♀️